hrjobcontract/js/test/controllers/modal/modal-change-reason_test.js

```diff
@@ -0,0 +1,116 @@
+define([
+    'common/moment',
+    'common/angularMocks',
+    'job-contract/app'
+], function(moment) {
+  'use strict';
+
+  describe('ModalChangeReasonCtrl', function() {
+    var $q, $rootScope, $scope, $controller, ctrl, modalInstanceSpy, ContractRevisionServiceSpy;
+
+    beforeEach(module('hrjc', 'job-contract.templates'));
```

Comment:
i think we are doing controller testing here so ps include the controller module only